apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -493,6 +508,62 @@ class CensusForm extends Component {
               </span>
             </label>
           </div>
+
+          {showExistingInaccuracy && (
+            <div>
+              <div style={{marginTop: 20, marginLeft: 38}}>
```

Comment:
Since we're using this style twice, here and line 498, maybe pull out of inline?